libs/form/src/next/datepicker/vl-datepicker.component.ts

```diff
@@ -19,8 +18,13 @@ import { registerWebComponents, webComponent } from '@domg-wc/common-utilities';
 import { VlButtonInputAddon, VlIconElement } from '@domg-wc/elements';
 import { live } from 'lit/directives/live.js';
 import { FormControl, formControlDefaults } from '../form-control/form-control';
+import { maskValidator } from './validators';
+import { CleaveInstance, MaskOptions } from '../../models/cleave.model';
+import Cleave from 'cleave.js';
+import { createDateMask, createTimeMask } from './masks';
 
 export const DATEPICKER_TYPES = ['date', 'range', 'time', 'date-time'] as const;
+export const dateRangeSeparator = ' tot en met ';
```

Comment:
Waarom wordt deze geëxporteerd?

Reply:
Dit is premature optimisation: het idee was dat de afnemer deze constante kon importeren wanneer nuttig maar met opkomend ticket waar we voor date range ook aanbieden als array van 2 dates is dit YAGNI. Ik pas het aan